### ISSUE TITLE
Unable to install MSMQ from windows features on windows 11

### ISSUE THREAD

**Issue by surajsugathan** [utmapp/UTM#6320]
On the UTM version i have with the simple setup of windows 11 configured, from the Windows features, I am unable to work with "Message queueing MSMQ activation" as each time it installs it, it asks for the restart(which is normal), but after the restart, its clear that the feature is not installed yet and doing the same process again, doesn't solve it.

**Configuration**  
* UTM Version: 4.4.4
* macOS Version: Sonoma 14.4.1
* Mac Chip : M2 Pro 
[Info.plist.zip](https://github.com/utmapp/UTM/files/15181348/Info.plist.zip)

**Comment by PAN740623** [comment]
XIAOYING PAN

**Comment by surajsugathan** [comment]
Found a post related to ARM64 machines having win 11 will not have MSMQ support from Microsoft
Link here: [https://devblogs.microsoft.com/dotnet/announcing-dotnet-framework-481/#:~:text=Message%20Queuing%20(MSMQ)%20Activation%20will%20remain%20disabled%20because%20MSMQ%20is%20not%20available%20on%20Windows%2011%20for%20ARM64%20client%20machines]

**Comment by sndbbbl** [comment]
Yes, I also read on many threads that ARM64 is not supporting MSMQ. I wait for this to happen since a long time. Have no clue why it is not supported...
